results/similar_developers_for_spadini.davide@gmail.com.json

```diff
@@ -0,0 +1,17 @@
+{
```

Comment:
ideally it should have some explanation why - top words for each developer, top repositories for each developer, top languages.